templates/demo-store/app/routes/($lang)/cart.tsx

```diff
@@ -71,6 +69,7 @@ export async function action({request, context}: ActionArgs) {
 
       break;
     case CartAction.REMOVE_FROM_CART:
+      invariant(cartId, 'Missing cartId');
```

Comment:
@scottdixon cart was complaining here because when you parsed the cookie with `getCartId` the function would have `string | undefined` union. 

Then some of the actions below required to have a string. So instead I just did a check with `invariant` if the cartId actually exists.